boot/freeldr/freeldr/include/ntldr/winldr.h

```diff
@@ -17,6 +17,31 @@
 // Some definitions
 
 #include <pshpack1.h>
+typedef struct /* ACPI Standard Header */
+{
+    CHAR Signature[4];
+    ULONG Length;
+    UCHAR Revision;
+    UCHAR Checksum;
+    CHAR OEMID[6];
+    CHAR OEMTableID[8];
+    ULONG OEMRevision;
+    ULONG CreatorID;
+    ULONG CreatorRevision;
+} ACPISDTHeader, *PACPISDTHeader;
```

Comment:
In case these structures already exist elsewhere (header of hal, or acpica...) I would _strongly_ recommend using the existing names, instead of new made-up ones.

Reply:
see the new name.